### ISSUE TITLE
Link error on Expo managed project

### ISSUE THREAD

**Issue by Spharian** [pusher/pusher-websocket-react-native#42]
After using the `expo install @pusher/pusher-websocket-react-native` command on an expo managed project, I get the following error:

The package 'pusher-websocket-react-native' doesn't seem to be linked. Make sure:

- You have run 'pod install'
- You rebuilt the app after installing the package
- You are not using Expo managed workflow

Pod install isn't available in a managed project, is it a bug or I am missing something?

Expo version 45.0.0
pusher-websocket-react-native version 1.1.0

**Comment by benw-pusher** [comment]
Are you able to run this using the prebuild command as part of the run process? See https://docs.expo.dev/workflow/customizing/#adding-custom-native-code-with-development-builds

**Comment by Spharian** [comment]
> Are you able to run this using the prebuild command as part of the run process? See https://docs.expo.dev/workflow/customizing/#adding-custom-native-code-with-development-builds

Well I tried, but I get the following error:

```
Something went wrong running `pod install` in the `ios` directory.
```

I think this doesn't work because I'm on managed expo workflow, no?

**Comment by Spharian** [comment]
Solved by forcing my deploymentTarget to 13.0.

**Comment by itzmail** [comment]
> Solved by forcing my deploymentTarget to 13.0.

how to set deploymentTarget to 13.0?
